source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,251 @@
+-----------------------------
+
+Migrating your App's Sync mode involves changes that affect both client apps and
+your App's backend. You should be aware of the side effects and plan ahead.
+
+- How long a migration takes is directly proportional to the amount of data that
+  needs to be migrated. The more data, the longer it will take to migrate.
+- Migrations can be :ref:`canceled <realm-sync-migrate-cancel>` during the 
+  migration or :ref:`reverted <realm-sync-migrate-revert>` within a certain 
+  timeframe after the migration is finished.
+- Progress notifications will not work after migration because Flexible Sync does
+  not support them.
+- Client apps that use an SDK with with a realm-core version lower than TBD will
+  experience a client reset after migration. You should use a 
+  :ref:`client reset <handle-a-client-reset>` handler with recovery enabled.
+
+.. _realm-sync-migrate-stages:
+
+Migration Stages
+~~~~~~~~~~~~~~~~
+
+There are three stages to a Sync mode migration. You can see your migration's
+stage in the App Services UI.
+
+#. **Syncing Partition-Based Sync metadata** 
+   
+   At this stage, client apps can connect to your App's backend, but local writes 
+   will not sync. These writes will be lost if you don't have a 
+   :ref:`client reset handler <handle-a-client-reset>` to recover unsynced changes.
+	 
+   Clients can also receive updates to synced collections, but they won't
+   be able to receive upload acknowledgements.
+
+#. **Building Flexible Sync metadata** 
+   
+   Clients cannot connect at this stage. The duration of this stage is directly
+   proportional to the amount of data in synced collections.
+
+#. **Completed migration** 
+
+   :guilabel:`Manage Sync Migration` appears on the Device Sync page if the 
+   migration hasn't been committed or reverted.
+
+Migrate Partition-Based Sync to Flexible Sync
+---------------------------------------------
+
+After beginning migration from Partition-Based Sync to Flexible Sync, the process
+is mostly automatic. However, you can control important parts of the migration.
+
+During a migration, you can cancel it. After a migration, you can revert it 
+or manually commit the migration.
+
+Begin Migration
+~~~~~~~~~~~~~~~
+
+Before you begin a migration, make sure you're aware of the
+:ref:`migration stages <realm-sync-migrate-stages>` and
+:ref:`effects <realm-sync-migrate-what-to-know>`.
+
+During migration, client apps may experience side effects, most of which are similar
+to what happens if you :ref:`terminate <terminating-realm-sync>` and re-enable
+Device Sync. These are some additional side effects of migration:
+
+- In the App Services UI, the Rules and Schema pages will be read-only.
+- Your App will experience a period of read-only Sync and then a 
+  period of downtime.
+- If Partition-Based Sync :ref:`client max offline time <client-maximum-offline-time>`
+  is unset, the new Flexible Sync configuration will set it to 30 days.
```

Comment:
UPDATE: we're removing the "client max offline time" setting for PBS apps, so we decided that for a migration, we will always set the flexible sync max offline time to 30 days regardless of their old PBS setting